File: app/src/main/java/com/micromos/knpmobile/CustomAutoCompleteTextView.java
package com.micromos.knpmobile;

import android.content.Context;
import android.util.AttributeSet;

public class CustomAutoCompleteTextView extends androidx.appcompat.widget.AppCompatAutoCompleteTextView {

    public CustomAutoCompleteTextView(Context context) {
        super(context);
    }

    public CustomAutoCompleteTextView(Context context, AttributeSet attrs) {
        super(context, attrs);
    }


    public CustomAutoCompleteTextView(Context context, AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
    }


    @Override
    public void onFilterComplete(int count) {
    }

    @Override
    protected void performFiltering(CharSequence text, int keyCode) {
        super.performFiltering(text, keyCode);
    }

    @Override
    public boolean enoughToFilter() {
        return true;
    }
}

File: app/src/main/java/com/micromos/knpmobile/CustomAutoCompleteTextView.kt
package com.micromos.knpmobile

import android.content.Context
import android.os.Handler
import android.util.AttributeSet
import android.util.Log
import android.widget.ArrayAdapter
import android.widget.AutoCompleteTextView
import androidx.appcompat.widget.AppCompatAutoCompleteTextView

class CustomAutoCompleteTextView : AppCompatAutoCompleteTextView {
    constructor(context: Context?) : super(context!!) {}
    constructor(context: Context?, attrs: AttributeSet?) : super(
        context!!, attrs
    ) {
    }

    constructor(context: Context?, attrs: AttributeSet?, defStyleAttr: Int) : super(
        context!!, attrs, defStyleAttr
    ) {
    }

    override fun onFilterComplete(count: Int) {}
    override fun performFiltering(text: CharSequence, keyCode: Int) {
        super.performFiltering(text, keyCode)
    }

    override fun enoughToFilter(): Boolean {
        return true
    }
}